circle.yml

```diff
@@ -6,6 +8,11 @@ machine:
 dependencies:
     override:
         - echo "Dummy override, so no Circle dependencies execute"
+        - if [[ -e ~/docker/image.tar ]]; then docker load -i ~/docker/image.tar; fi
```

Comment:
Is this to avoid docker pulling images from the hub on each test run?  Not sure I follow why the built container is saved and reloaded here.